Sources/TelemetryClient/LogHandler.swift

```diff
@@ -29,7 +30,16 @@ public struct LogHandler {
 
     public static var stdout = { logLevel in
         LogHandler(logLevel: logLevel) { level, message in
-            print("[TelemetryDeck: \(level.description)] \(message)")
+            if #available(macOS 11.0, *) {
```

Comment:
The availability check here only checks for macOS, but this lib supports other platforms as well. Those should be checked, too. Consider also Linux/Windows platforms.